### PR TITLE
Add banner when there is a problem with credits

### LINKED ISSUE #9755: add red banner at top of page when there is a problem with credits
A banner should show up on the top of every page in the cloud UI when the workspace's credit usage is in a bad state.

See https://github.com/airbytehq/airbyte-cloud/pull/847 for the endpoint changes that expose the credit statuses.

Here's the copy for each case:
- `negative_within_grace_period` -> `Your workspace is out of credits! Please add more credits to make sure your syncs keep running!`
- `negative_beyond_grace_period` -> `No syncs can run because this workspace is out of credits! Add more credits to resume syncing.`
- `negative_max_threshold` -> `No syncs can run because this workspace is out of credits! Add more credits to resume syncing.`

All of the banners should link to the billing credits page.

### PR DESCRIPTION
**Pull request by juliachvyrova**
## What
Add red banner at top of page when there is a problem with credits

Снимок экрана 2022-02-18 в 16 14 38

Closes #9755

### CODE REVIEW

**Review by jrhizor — APPROVED** [review]

File: airbyte-webapp/src/packages/cloud/views/layout/MainView/components/CreditsProblemBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import { CreditStatus } from "packages/cloud/lib/domain/cloudWorkspaces/types";
+import { Link } from "components/Link";
+import { CloudRoutes } from "packages/cloud/cloudRoutes";
+
+const Container = styled.div`
+  height: 30px;
+  width: 100%;
+  background: ${({ theme }) => theme.redColor};
+  color: ${({ theme }) => theme.whiteColor};
```
Comment (timroes) [inline airbyte-webapp/src/packages/cloud/views/layout/MainView/components/CreditsProblemBanner.tsx:13]: We need to use black text on that background color to achieve the proper (see https://color.review/check/FFFFFF-FF6A4D).

File: airbyte-webapp/src/packages/cloud/views/layout/MainView/components/CreditsProblemBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  text-align: center;
+  position: fixed;
+  z-index: 3;
+  font-size: 12px;
+  line-height: 30px;
+`;
+const CreditsLink = styled(Link)`
+  color: ${({ theme }) => theme.whiteColor};
```
Comment (timroes) [inline airbyte-webapp/src/packages/cloud/views/layout/MainView/components/CreditsProblemBanner.tsx:21]: Same here, should use black text.

**Review by timroes — CHANGES_REQUESTED** [review]
Some contrast ratio issue and some typing correction. Besides that looks good.

**Review by timroes — APPROVED** [review]
Code LGTM, haven't tested locally.

**Pull request merged**